eta/core/image.py

```diff
@@ -184,16 +184,15 @@ def filter_by_schema(self, schema):
         self.attrs.filter_by_schema(schema.attrs)
         self.objects.filter_by_schema(schema)
 
-    def remove_objects_without_attrs(self, object_labels_to_filter=None):
-        '''Removes DetectedObjects from this object that do not have attributes.
+    def remove_objects_without_attrs(self, labels=None):
+        '''Removes DetectedObjects from this instance that do not have
+        attributes.
 
         Args:
-            object_labels_to_filter: a list of DetectedObject.label strings.
-                DetectedObject.labels not in this list are not removed even if
-                they have no attributes. If None, all object labels are
-                filtered.
+            labels: an optional list of object labels to which to restrict
+                attention when filtering. By default, all objects are processed
```

Comment:
I'm not keen on this description of the arg. I want to make it clear what each element of the list is, which is two things:
1) a string
2) a value that will be matched/compared against a `DetectedObject.label`
I'm indifferent to the exact syntax by which we convey that, but I think it makes this documentation a lot easier if these two snippets of information are conveyed.

Reply:
do what you will. PR is approved for you